state/execution.go

```diff
@@ -302,9 +310,18 @@ func (blockExec *BlockExecutor) Commit(
 	)
 	updateMempoolEndTime := time.Now().UnixNano()
 
-	updateMempoolTimeMs := float64(updateMempoolEndTime-updateMempoolStartTime) / 1000000
+	updateMempoolTimeMs := float64((updateMempoolEndTime-updateMempoolStartTime) / 1000000) / 1000.0
```

Comment:
Why did you change it to second?

Reply:
Originally it was showing milliseconds in integer, but now it's showing seconds in float. I'll change the variable name to updateMempoolTime.